src/pyhf/optimize/opt_minuit.py

```diff
@@ -45,20 +45,23 @@ def f(pars):
         )
         return mm
 
-    def unconstrained_bestfit(self, objective, data, pdf, init_pars, par_bounds):
-        # The Global Fit
-        mm = self._make_minuit(objective, data, pdf, init_pars, par_bounds)
+    def minimize(self, objective, data, pdf, init_pars, par_bounds, fixed_vals=None):
```

Comment:
call this `bestfit` or `fit` maybe? Or maybe we change the other calls to `unconstrained_minimization` and `constrained_minimization`?

Reply:
I would review this later. generally I think there should be a `pyhf.stats.fit` method which internally uses `pyhf.optimizer.minimize` and remove `unconstrained_bestfit` and `constrained_bestfit` long-term

Reply:
I like that.